Formula/libvisual.rb

```diff
@@ -0,0 +1,76 @@
+class Libvisual < Formula
+  desc "Audio Visualization tool and library"
+  homepage "https://github.com/Libvisual/libvisual"
+  url "https://github.com/Libvisual/libvisual/releases/download/libvisual-0.4.2/libvisual-0.4.2.tar.gz"
+  sha256 "63085fd9835c42c9399ea6bb13a7ebd4b1547ace75c4595ce8e9759512bd998a"
+  license all_of: ["GPL-2.0-or-later", "GPL-3.0-or-later", "LGPL-2.1-or-later"]
+
+  depends_on "pkg-config" => :build
+  depends_on "xorg-server" => :test
+
+  depends_on "jack"
+  depends_on "portaudio"
+  depends_on "sdl12-compat"
+
+  uses_from_macos "bison" => :build
+
+  on_linux do
+    depends_on "alsa-lib"
+    depends_on "mesa"
+    depends_on "mesa-glu"
+    depends_on "pulseaudio"
+  end
+
+  resource "libvisual-plugins" do
+    url "https://github.com/Libvisual/libvisual/releases/download/libvisual-plugins-0.4.2/libvisual-plugins-0.4.2.tar.gz"
+    sha256 "55988403682b180d0de5e6082f804f3cf066d9a08e887b10eb6a315eb40d9f87"
+  end
```

Comment:
Is this always the same version as `libvisual`? If so, I suggest doing something like this in the `test` block:

```ruby
assert equal version, resource("libvisual-plugins").version
```